jdaviz/configs/cubeviz/plugins/viewers.py

```diff
@@ -167,6 +207,10 @@ def active_image_layer(self):
 
         return visible_layers[-1]
 
+    def _on_global_display_unit_changed(self, msg):
+        if 'slice_values' in self.__dict__:
+            del self.__dict__['slice_values']
```

Comment:
This seems very drastic. Maybe add a comment to explain why it is needed? Thanks!